### PR TITLE
Adding more topics to py_topics.yaml

### PR DESCRIPTION
**Pull request by blankRiot96**
## Relevant Issues
There was a major problem with lancebot repeating the same questions in python-general, so i decided to make a pr to make more :) 
Permission aquired from -https://discord.com/channels/267624335836053506/635950537262759947/850418614468345856

## Description
 Added more questions to lancebot 

## Did you:
- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/resources/evergreen/py_topics.yaml [hunk 1]
```diff
@@ -23,6 +23,19 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Is python your first programming language, if not, what is?
+    - What's your favourite aspect of python development? (Backend web dev/game dev/machine learning and ai ect)
+    - In what ways has python discord helped you with python?
+    - What part of python is currently dieing?
```
Comment (kosayoda) [inline bot/resources/evergreen/py_topics.yaml:30]:
```suggestion
```
I'm not a fan of this question, would you mind either rewording it or removing it?

**Review by kosayoda — CHANGES_REQUESTED** [review]
Thanks for the PR, can you please capitalize `Python` and `Python Discord`?

**Review by eivl — CHANGES_REQUESTED** [review]

**Comment by Xithrius** [comment]
@blankRiot96 Will you be able to complete the requested changes?

File: bot/resources/evergreen/py_topics.yaml [hunk 2]
```diff
@@ -23,6 +23,19 @@
...
+    - Are you currently a student, or using python professionally?
```
Comment (dawnofmidnight) [inline bot/resources/evergreen/py_topics.yaml:31]: Mentioning something about hobby programmers would be nice here, as there are many of those.

**Review by dawnofmidnight — CHANGES_REQUESTED** [review]

**Comment by Xithrius** [comment]
@blankRiot96 will you be able to finish of this PR? If so, please let me know.

**Comment by blankRiot96** [comment]
> @blankRiot96 will you be able to finish of this PR? If so, please let me know.

probably not, as i was banned from the server 
do what you want with the questions

**Comment by Xithrius** [comment]
I see.

**Comment by Xithrius** [comment]
Being continued in https://github.com/python-discord/sir-lancebot/pull/792.

**Pull request closed without merging**